src/tutorials/cloudery/index.md

```diff
@@ -0,0 +1,67 @@
+# Cloudery
+
+This documentation explain how to use our Cloudery API to create and manager our own Cozy.
+Code example is provided for Ruby language, but this can be easily ported to any other langage, there is no real dependency to the language by itself.
+
+Full API documentation is provided [here](https://cloudery.cozycloud.cc/api/) and this documentation is only about some standard usages examples.
+
+## Creating a Cozy
+
+### From web
+
+Send your user on `/v2/{partner}/onboard`. You can customize some onboarding parameters with URL query, see [the documentation](https://cloudery.cozycloud.cc/api/#tag/instances/paths/~1{partner}~1onboard/get).
+Our Cloudery will onboard the user throught differents steps to gather required informations and then create the instance.
+
+![email](email.png)
+![FQDN](fqdn.png)
+![terms](terms.png)
+![confirm](confirm.png)
+
+Your user will received an email at the end, with an activation link to click to go on the initial passphrase definition.
+
+![validate](validate.png)
+![passphrase](passphrase.png)
+
+### From API called by a client app.
+
+You can create an instance directly from our API.
+
+Call the [`POST /api/public/instances`](https://cloudery.cozycloud.cc/api/#tag/public/paths/~1public~1instances/post) route, then poll the [`GET /api/public/workflows/{id}`](https://cloudery.cozycloud.cc/api/#tag/public/paths/~1v1~1public~1workflows~1{uuid}/get) til the end of the creation.
+You need to have a public token to authenticate you on those endpoints.
+
+```python
+import requests
+
+token = 'xxxx'
+headers = { 'Authorization': 'Bearer ' + token }
+
+base_url = 'https://manager.cozycloud.cc/api'
+response = requests.post(base_url + '/v1/instances', headers=headers, json={
+                            'offer': 'some_offer',
+                            'email': 'foo@example.org',
+                            'slug': 'foo', 'domain': 'example.org'
+                        })
+response = response.json()
```

Comment:
Before calling `response.json()` you should check for reponse.status_code.
Then response.json() should be enclosed within `try ... catch JSONDecodeError...` in case response is not a json object (reverse proxy error by example.

Reply:
I think it is not our responsibility to manage errors and here we should only provide core example.
Integrators must know the language and libraries they use and must adapt snippets

And almost all lines of this snippet can throw error or behave badly. From `requests.post` with network error (`NewConnectionError` for example), `response['workflow']` if JSON correct but field not present, real code to cover all the errors is damn more complexe than just catching JSON trouble.